### ISSUE TITLE
ConfusionMatrix - Plot numbers inside cells

### ISSUE THREAD

**Issue by scls19fr** [pandas-ml/pandas-ml#92]
following https://github.com/pandas-ml/pandas_confusion/issues/2

http://stackoverflow.com/questions/5821125/how-to-plot-confusion-matrix-with-string-axis-rather-than-integer-in-python

![capture d ecran 2015-10-31 a 09 19 23](https://cloud.githubusercontent.com/assets/109167/10862634/8340e9a4-7fb0-11e5-8212-d259f5e52cdb.png)

see also Seaborn Heatmap code https://stanford.edu/~mwaskom/software/seaborn/generated/seaborn.heatmap.html

**Comment by sinhrks** [comment]
including other issues, it may be easier to impl our own plot method, rather than using seaborn?

**Comment by scls19fr** [comment]
Yes but what I'm saying here is that we can have a look at seaborn code for this
